packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/index-new.ts

```diff
@@ -94,6 +94,7 @@ import processedIcons_numberlist from './icons-number-list'
 import processedIcons_notificationoff from './icons-notification-off'
```

Comment:
not related to this PR - think that we should remove this file, as there is no any utility in it now.